RealmSwift/Object.swift

```diff
@@ -244,4 +287,11 @@ public class ObjectUtil : NSObject {
 
         return properties
     }
+
+    @objc private class func initializeListProperty(object: RLMObjectBase?, property: RLMProperty?, array: RLMArray?) {
+        if let object = object as? Object {
+            let list = object[property!.name]! as RLMListBase
```

Comment:
There are 3 operations in this line that could fail (plus the optional binding above). Throwing an exception if any of these fail will help users (or us) figure out what might have gone wrong.